### ISSUE TITLE
can we combine the original dataset and re-divided to perform cross-validation?

### ISSUE THREAD

**Issue by SIPUNK** [github/CodeSearchNet#244]
how do you divide the training/testing/validation set, I don't seem to find the basis for the division, only the ratio. If we want to perform cross-validation, can we combine the original training set, test set, and validation set and re-divided.
